eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -490,6 +490,11 @@
         "old": ".*",
         "annotationType": "com\\.azure\\.core\\.annotation\\.Service(Method|Client)",
         "justification": "These are SDK metadata annotations and don't affect runtime behavior."
+      },
+      {
+        "code": "java.annotation.attributeValueChanged",
+        "new": "@interface com.azure.core.annotation.JsonFlatten",
+        "justification": "Changes made the annotation more permissive on the target types."
```

Comment:
Do we want to have a follow-up issue for tracking the removal of this exception after next GA release so that these exceptions do not automatically permit future changes? (applies to all exceptions we have in this file)

Reply:
I think this would be a good quarterly/bi-annual task to clean up RevApi, and Checkstyle and Spotbug, suppressions as the repository changes.